NebulaHost/PacketProcessors/Logistics/StationUIBroadcaster.cs

```diff
@@ -22,18 +22,37 @@ public StationUIBroadcaster()
         }
         public void ProcessPacket(StationUI packet, NebulaConnection conn)
         {
-            Player player = playerManager.GetPlayer(conn);
             // if a user adds/removes a ship, drone or warper or changes max power input broadcast to everyone.
-            if (player != null && StationUIManager.UpdateCooldown == 0 &&
+            if (StationUIManager.UpdateCooldown == 0 &&
                 (packet.SettingIndex == StationUI.EUISettings.MaxChargePower
                  || packet.SettingIndex == StationUI.EUISettings.SetDroneCount
                  || packet.SettingIndex == StationUI.EUISettings.SetShipCount
                  || packet.SettingIndex == StationUI.EUISettings.SetWarperCount
-                 || packet.SettingIndex == StationUI.EUISettings.AddOrRemoveItemFromStorageRequest
-                 || packet.SettingIndex == StationUI.EUISettings.AddOrRemoveItemFromStorageResponse)
+                 )
                 )
             {
-                playerManager.SendPacketToAllPlayers(packet);
+                // this is the SendPacketToAllPlayers() logic but we need to set the mimic flag here.
+                using (playerManager.GetConnectedPlayers(out var connectedPlayers))
+                {
+                    foreach (var kvp in connectedPlayers)
+                    {
+                        Player p = kvp.Value;
+                        if (p.Connection == conn)
+                        {
+                            packet.ShouldMimic = true;
```

Comment:
This value is never reset? So, if we have 3 players:
```
p1
p2 == conn
p3
```

Both p2 and p3 will have ShouldMimic = true, but doesn't look right.